bin/cli.js

```diff
@@ -81,6 +99,11 @@ function rewrite(cb){
   walkdir(dir,function(p,stat){
     if(p.substr(p.length-3) !== '.js') return
     if(~p.indexOf('.git') || ~p.indexOf('node_modules')) return
+    for (var i = 0; i < ignorePaths.length; i++){
```

Comment:
this could be `if(ignorePaths.indexOf(p) === 0) return;` instead of the for right?